### PR TITLE
Faq section updated

### PR DESCRIPTION
**Pull request by Pragati-3003**
- Updated the fag section of the website
- Deep Clean the `resource.html` and `resource.css` and `styles.css`.

### CODE REVIEW

File: resource.css [hunk 1]
```diff
@@ -440,6 +463,11 @@ section h5 {
   margin-right: 4px;
   padding: 5px;
 }*/
+
+/* .imge{
+  margin: 5px;
+  box-shadow: 2px 2px 8px rgba(253, 208, 126, 0.342)
+} */
```
Comment (rohanbabbar04) [inline resource.css:470]: Remove the comments...

**Review by rohanbabbar04 — COMMENTED** [review]
Just remove the commented codes, rest looks good :)

**Comment by Pragati-3003** [comment]
> @Pragati-3003 is it good to go??

Ya , now you can merge , I have removed all the unnecessary comments.

**Review by rohanbabbar04 — APPROVED** [review]

**Comment by rohanbabbar04** [comment]
Thanks :)

**Pull request merged**